src/02 - Object types.md

```diff
@@ -29,559 +32,560 @@ The following notation is used in the tables below:
-|      | ref    | refMappe                          |          |             |                |  |      |      |   x   |      |       x       | <p>&bull; references **Mappe** or **Saksmappe**</p> <p>&bull; may not be supported by all systems or partially supported</p>               |                                                                                                             |
-|      | ref    | refRegistrering                   |          |             |                |  |      |      |   x   |      |       x       | <p>&bull; references **Basisregistrering** or **Journalpost** </p><p>&bull; may not be supported by all systems or partially supported</p> | <p>&bull; references to **Basisregistrering** and **Journalpost** not currently supported in Documaster</p> |
+| Field no. | Type   | Field                             | Not null | Has default | Code list      |  | Save | View | Query | Sort | Link | Unlink | Comment                                                                                                                                                                        |
+|:----------|:-------|:----------------------------------|:--------:|:-----------:|:---------------|:-|:----:|:----:|:-----:|:----:|:----:|:------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|           | string | id                                |    x     |      x      |                |  |  sm  |  x   |   x   |  x   |      |        |                                                                                                                                                                                |
+|           | string | version                           |    x     |      x      |                |  |  sm  |  x   |   x   |      |      |        |                                                                                                                                                                                |
+|           |        |                                   |          |             |                |  |      |      |       |      |      |        |                                                                                                                                                                                |
+| M001      | string | uuid                              |    x     |      x      |                |  |  cw  |  x   |   x   |      |      |        |                                                                                                                                                                                |
```

Comment:
Sorting on `uuid`s is supported for all objects.

Reply:
Fixed